### ISSUE TITLE
 Create an NVDA touch gesture to mimic the windows plus H to toggle dictation on and off  when a keyboard is present

### ISSUE THREAD

**Issue by Kanwar5** [nvaccess/nvda#11854]
Please consider implementing a gesture for the touch screen that would allow a user to toggle dictation on and off,  when a keyboard is present on the screen, in the same way that pressing Windows and H key combination will allow the user to toggle the dictation feature on and off.

**Comment by josephsl** [comment]
Hi, is this something Narrator provides? Can you also try using microphone button found on touch keyboard? I think this is something Enhanced Touch Gestures add-on should take care of if this is accepted (i.e. I’ll take care of it as I’m the author of that add-on). Thanks.

**Comment by leonardder** [comment]
I think you should be able to do this yourself using the input gestures dialog. See the 12.2.3. Input Gestures in the NVDA user guide.

**Comment by josephsl** [comment]
Hi, I think not, because keyboard command emulation is limited to braille displays at the moment. I brought up touch keyboard idea because it comes with a microphone button last time I checked. Thanks.

**Comment by Kanwar5** [comment]
As far as I know, narrator doesn't provide this capability, there is a way to toggle dictation on and off through the keyboard by pressing Windows key and H together.
There is also a button on the touch keyboard but these buttons are small and can easily be missed, and when doing work which involves both dictation and typing, I feel it would be useful to have a gesture that could mimic this feature, iOS for example has a two finger double tap to toggle dictation on and off


Khalid Anwar
Please call 07497613820 or email AnwarKhalid850@gmail.com

**Comment by josephsl** [comment]
Hi, rather than adding this to NVDA directly, I think it would be best to do it as part of Enhanced Touch Gestures add-on (I’ll think about that next year). Note that Windows+H emulation will be available on Windows 10 Version 1809 (October 2018 Update) and later, or if justified, Version 1909 (November 2019 Update) and later due to Microsoft’s support policy. Thanks.

**Comment by Kanwar5** [comment]
Excellent,
Thank you for considering the suggestion and hope you have a nice day.



Khalid Anwar
Please call 07497613820 or email AnwarKhalid850@gmail.com

**Comment by Adriani90** [comment]
It seems Windows+h emulation is now possible natively in Windows as indicated by @josephsl in https://github.com/nvaccess/nvda/issues/11854#issuecomment-731975275.

I am closing as works for me. If you are still having this issue, please comment and we can reopen.
